apps/system/utils/utils_ps.c

```diff
@@ -128,17 +128,19 @@ static void ps_print_values(char *buf, void *arg)
 		return;
 	}
 
+	printf("%5s | %4s | %4s | %7s | %c%c | %8s", stat_info[PROC_STAT_PID], stat_info[PROC_STAT_PRIORITY], \
+		flags & TCB_FLAG_ROUND_ROBIN ? "RR  " : "FIFO", utils_ttypenames[(flags & TCB_FLAG_TTYPE_MASK) >> TCB_FLAG_TTYPE_SHIFT], \
+		flags & TCB_FLAG_NONCANCELABLE ? 'N' : ' ', flags & TCB_FLAG_CANCEL_PENDING ? 'P' : ' ', \
+		utils_statenames[state]);
+
 #ifdef CONFIG_SMP
 	cpu_idx = atoi(stat_info[PROC_STAT_CPU]);
 	if (cpu_idx != cpu && cpu != CONFIG_SMP_NCPUS) {
 		return;
```

Comment:
What happen if we meet this return?
Just `return` is enough? Could you verify that?

Reply:
Then, how about checking this at very first?
Even others have `return`, they are error cases (I mean it's difficult to meet).
but this is a normal case (easy to meet). Adjusting the location will be good I think,

Reply:
done that already